### PR TITLE
Release 2.7.2-beta2

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming second beta of the v2.7.2 series? Well you've just found it, congratulations!

## Release checklist

- [x] ~~Check to see if there are any errors in Sentry that have only occurred since the last production release~~
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

**Comment by tidy-dev** [comment]
No feature flag changes since last beta1.
No new metrics either.

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.7.2-beta2": [
+      "[Added] User can expand diff hunks! - #11902",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Expand diffs above and below the changes to get more context in other parts of files - #11902",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] User could not undo cherry-pick onto a remote branch - #11921",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Undo cherry-picking onto a remote branch now works as expected - #11921",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+      "[Improved] User can end drag by pressing escape - #11925",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Cancel drag operation with escape key - #11925",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,12 @@
...
+      "[Improved] User can cherry-pick onto a pull request - #11922",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Cherry-pick commits onto a pull request - #11922",
```

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,12 @@
...
+      "[Improved] Windows users whose system language is not English can change spellcheck language to English - #11589"
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Option to change the spellcheck language to English for Windows users whose system language is not English - #11589"
```

**Review by billygriffin — CHANGES_REQUESTED** [review]
Thanks @tidy-dev! I reworded the release notes to make them a bit clearer.

File: changelog.json [hunk 6]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.7.2-beta2": [
+      "[Added] Expand diffs above and below the changes to get more context in other parts of files - #11902",
+      "[Fixed] Undo cherry-picking onto a remote branch now works as expected - #11921",
+      "[Improved] Cancel drag operation with escape key - #11925",
+      "[Improved] Cherry-pick commits onto a pull request - #11922",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Cherry-pick commits onto a pull request with drag and drop - #11922",
```

**Review by billygriffin — APPROVED** [review]
🎉

**Pull request merged**